.github/CODEOWNERS

```diff
@@ -32,3 +32,9 @@ Cargo.toml @didier-wenzek @albinsuresh @jarhodes314 @rina23q
 
 # Python
 *.py @reubenmiller @albinsuresh @jarhodes314
+
+# Robot Framework
+*.robot @gligorisaev
```

Comment:
I don't think this is necessary as the `/tests` rule already allows everyone to review the tests. Or what is the reasoning for having an explicit name in this case?

Reply:
I can confirm there is something weird: Gligor is not added as a reviewer on PRs with updates on `/tests` despite being listed as a contributor. This works only if the very first version of the PR impacts some test. However, I would be surprised if an extra rule solves the issue.

Reply:
The purpose is to add @gligorisaev to reviewers of PRs that changes doc and robot tests. 

> I can confirm there is something weird: Gligor is not added as a reviewer on PRs with updates on /tests despite being listed as a contributor. This works only if the very first version of the PR impacts some test.

For example, @gligorisaev was not added to reviewers of this PR https://github.com/thin-edge/thin-edge.io/pull/2826. The PR has a change on a `.robot` file, although you can see Albin and James are added because they are "CODEOWNERS".

I can't confirm it easily (requiring my organization + repo setup, I'm too lazy to do. If someone request it to me, I'll do.), but I have a hypothesis, maybe we need to add his name explicitly. Then I want to see how it works.